src/components/UserDashboard/index.js

```diff
@@ -49,10 +50,13 @@ export default compose(
     <div className={userDashboardContent}>
       <h1 className={dashboardTitle}>My Dashboard</h1>
       {/* [NEXT] SizeProvider here is the only usage of 'react-sizeme' */}
-      <Row gutter={[30, 30]} align={'top'} style={{paddingLeft: 15, paddingRight: 15, top:-15}}>
+      <Row gutter={[30, 30]} align={'top'} style={{ paddingLeft: 15, paddingRight: 15, top: -15 }}>
```

Comment:
if simple, I d suggest using css

Reply:
didn't catch that since I didn't touch that part, it was auto cleaned by the linter, will fix